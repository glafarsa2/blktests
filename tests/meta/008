#!/bin/bash
#
# Test device_requires().
#
# Copyright (C) 2017 Omar Sandoval
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

DESCRIPTION="skip in device_requires()"

device_requires() {
	SKIP_REASON="(╯°□°)╯︵ $TEST_DEV ┻━┻"
	return 1
}

test_device() {
	echo '¯\_(ツ)_/¯'
}
