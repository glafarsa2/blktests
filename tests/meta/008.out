Running meta/008
Test complete
